.github/workflows/update-registry.yml

```diff
@@ -0,0 +1,82 @@
+name: Update Registry
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Safe version with deployment type'
+        required: true
+        type: choice
+        options:
+          - v1.3.0-canonical
+          - v1.3.0-eip155
```

Comment:
Future feature request: might make sense to add a `v1.3.0` that does both `eip155` and `canonical` at the same time.